src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/NewApplicationPaymentsTest.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.bulkscan.endtoendtests;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.helper.Await;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.helper.StorageHelper;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.helper.ZipFileHelper;
+
+import static java.util.Collections.singletonList;
+
+public class NewApplicationPaymentsTest {
+
+    @Test
+    public void should_upload_blob_and_create_exception_record() throws Exception {
+        String zipFileName = ZipFileHelper.randomFileName();
+
+        // create zip file
+        var zipArchive = ZipFileHelper.createZipArchive(
+            singletonList("test-data/new-application-payments/1111002.pdf"),
+            "test-data/new-application-payments/metadata.json",
+            zipFileName
+        );
+
+        // upload zip file
+        StorageHelper.uploadZipFile("bulkscan", zipFileName, zipArchive);
```

Comment:
I'd say both comments can be removed. They'll literally methods names that are being called